bigquery/tests/unit/test__helpers.py

```diff
@@ -1481,6 +1528,81 @@ def test_to_api_repr_w_nested_struct(self):
         self.assertEqual(param.to_api_repr(), EXPECTED)
 
 
+class Test__query_param_from_api_repr(unittest.TestCase):
```

Comment:
Note: this is probably related to https://github.com/GoogleCloudPlatform/google-cloud-python/issues/3870